MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+https://github.com/MinaFoundation/mina-on-chain-voting/tree/main 
+https://github.com/MinaFoundation/govbot 
+- Governance Dashboards. The community is encouraged to experiment with different methods and tools to increase participation in the MIP process.
+https://minascan.io/mainnet/governance/MIPs 
+
+We propose to introduce new types of public debates:
+
+- During the expert part of the Review Phase: The MIP Facilitator will schedule, livestream and record open and public meetings between the MIP Author, the MIP Review Group and other stakeholders. The meeting minutes will be posted and shared in the designated channels for public consumption, allowing the community members who are interested to follow the process and contribute with comments. 
+- During the community part of the Review Phase: There will be a period of time for the community to carefully read and consider the MIP and the Review Group’s Risk Assessment. The MIP Facilitator will coordinate with the Mina Foundation to organize a community call where the MIP Author and the MIP Review Group can explain the MIP, including its risks, benefits and tradeoffs, in a way that is comprehensible to non-expert audiences and respond to community questions. Dates for the on-chain vote will be set, along with instructions on how to participate.
+
+### Changes in the Drafting and Review Phases
+
+**Drafting Phase - MIP Submission**
+
+During the Drafting phase, when the MIP Author submits a new MIP as a pull request in GitHub, a MIP Facilitator is assigned to assist the MIP Author, review the MIP and (if necessary, consult with the MIP Reviewers to) decide whether to approve or reject the pull request according to the following acceptance criteria:
+- The proposal is original or new.
+- The proposal is clearly articulated.
+- The proposal uses the MIP Template correctly.
+- The proposal is not missing any required parts.
+
+If the pull request is rejected, the MIP Facilitator will communicate the decision to the MIP Author explaining the reasons for the rejection and a list of changes that could be made for the proposal to be accepted, so that the MIP Author can make these changes.
+
+If the pull request is accepted, the MIP Facilitator:
+- Approves the MIP and assigns it a formal MIP number.
+- Changes the status of the MIP.
+- Merges in the PR.
+- Communicates the decision to the MIP author.
+
+**Review Phase - Part 1: Risk Assessment**
+We propose to create a pool of experts to review MIPs across the 3 categories. For each MIP, the Author reaches out to the pool of experts for the relevant category and sends a Request for Comments. To maintain independence, the Author does not choose who the Reviewers are. This MIP Review Group has a period to review the MIP and provide their feedback. The MIP Author can incorporate the feedback into the MIP until a final draft is completed. If the MIP Author does not incorporate the feedback, the MIP process still continues.
```

Comment:
Are reviewers self-selected and voluntary for any given MIP? While this states the Author does not choose, it is unclear how Reviewers are selected, if they are. A voluntary approach seems reasonable. Perhaps with the facilitator taking responsibility for notifying the group of insufficient participation.

Reply:
While it is said in the implementation plan, it should be clear her that if an expert is also an author, they are not counted as a Reviewer. Similarly, and Author may not be the Facilitator for their own MIP.

Reply:
I think this part of the process needs more clarity. In addition to the questions and notes raised by @jrwashburn above, there needs to be more clarity around how a MIP is determined to fit into the 3 categories, and whether a MIP can span across multiple categories. Eg: most economic changes have direct consensus considerations in the protocol that effect security. When and how do we ensure the protocol reviewer group is looped into the risk assessment of such a MIP? I can think of crossover examples like this for nearly each permutation of the proposed categories.

Reply:
You're right, we opted for a voluntary approach for reviewers at this stage and the authors should not be allowed to review or facilitate their own MIP to maintain impartiality. I’ll update this section for further clarity, thanks for your input!

Looking ahead and as the process evolves, we could explore additional mechanisms for selecting reviewers if necessary but for now, I consider this voluntary approach a solid foundation.